lib/amplitude_web.dart

```diff
@@ -40,6 +71,81 @@ class AmplitudeFlutterPlugin {
           return amplitude.logEvent(eventType, eventProperties);
           //TODO: support outOfSession
         }
+      case "logRevenue":
+        {
+          double price = args['price'] ?? 0;
+          int quantity = args['quantity'] ?? 1;
+          var productIdentifier = args['productIdentifier'] ?? null;
+          return amplitude.logRevenue(price, quantity, productIdentifier);
+        }
+      case "logRevenueAmount":
+        {
+          double amount = args['amount'] ?? 0;
+          return amplitude.logRevenue(amount, 1, null);
+        }
+      case "identify":
+        {
+          return true;
+        }
+      case "setGroup":
+        {
+          String groupType = args['groupType'];
+          String groupName = args['groupName'];
```

Comment:
Is our current flutter interface has group name string only? Noticed in JS surface a dynamic. I believe it is fine if Flutter interface only has String accepted.

Reply:
Take a quick look, seems that is `dynamic`.

Reply:
Good find. I change the type to dynamic.